Localize/lang/strings.json

```
@@ -782,6 +780,7 @@
   "_+JCU+Y.comment": "Tooltip for on-prem gateway connection checkbox",
   "_+W2vTp.comment": "Placeholder description for a newly inserted Email parameter",
   "_+W9fkq.comment": "Timezone value ",
+  "_+ZSBrq.comment": "Accessibility label",
   "_+dlOgj.comment": "Timezone value ",
   "_+g+mch.comment": "Label for description of custom multipartBody Function",
   "_+iMTt7.comment": "aria label description for check button",
```

What is an "aria label"? What's a "check button"?

```suggestion
  "_+iMTt7.comment": "Aria label description for check button",
```